source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+
+---
+title: 'PHP Core Roundup #9'
+layout: post
+tags:
+    - roundup
+author:
+  name: Ayesh Karunaratne
+  url: https://aye.sh
+published_at: 30 January 2023
+---
+
+The first for the year 2023, and the ninth in the series, this episode of PHP Core Roundup is full of exciting new developments in PHP and about PHP 8.2, the latest PHP version released last month! January was a busy month with many RFC proposals, discussions, votes, and some that are already included for the upcoming PHP 8.3 version that is scheduled for November 2023.
+
+> The PHP Foundation currently supports six part-time PHP contributors who work on maintenance and new features for PHP. Maintenance is not limited to fixing bugs, but also includes work to reduce technical debt, making life easier for everyone working on PHP. The contributors funded by the PHP Foundation collaborate with other contributors on code, documentation, and discussions.
+
+<br>
+We publish the posts on our website, and you can subscribe to a newsletter; You don’t necessarily have to be a PHP Foundation backer to follow _PHP Core Roundup_.
+
+{% include "newsletter.html" %}
+
+## PHP 8.2 Released
+
+PHP 8.2 is here! With type-system improvements, `readonly `classes, sensitive parameter redaction support, a new "`random`" extension, and several new features and improvements, PHP 8.2.0 was released on December 08th, followed by a bug-fix release 8.2.1 on January 04th.
```

Comment:
```suggestion
PHP 8.2 is here! With type-system improvements, `readonly `classes, sensitive parameter redaction support, a new "`random`" extension, and several new features and improvements, PHP 8.2.0 was released on December 08th, followed by a security-fix release 8.2.1 on January 05th.
```

8.2.1  is a [security release](https://github.com/php/web-php/blob/d9a32bf038306184236cd2336620f1143a819f18/include/version.inc#L25)